autoload.php

```diff
@@ -19,6 +19,12 @@
     require_once __DIR__ . '/config.php';
 }
 
+// If composer autoloaded context such as eZ Platform & legacy bridge, skip trying to autoload Zeta Components and such
+if ( class_exists( 'Composer\Autoload\ClassLoader', false ) )
```

Comment:
How is this different from what legacy bridge bundle does?

https://github.com/ezsystems/LegacyBridge/blob/master/bundle/EzPublishLegacyBundle.php#L43-L46

Reply:
Had forgot about that one.

It's more or less the same, so not needed then. Only difference is that this also detects usage in plain legacy. As in:
```php
require __DIR__ . '/vendor/autoload.php';

$ini = eZINI::instance();
echo $ini->variable( 'SiteAccessSettings', 'ForceVirtualHost' );
```

That said I'm unsure if allowing yet another way to do things is good. So I'll gladly remove it.

Reply:
But I probably need it in order to be able to do the section in composer:
```json
    "autoload": {
      "files": ["autoload.php"]
    },
```

Reply:
Okay. I'll test the patch tomorrow to see how it behaves.

Reply:
Btw, is there a point in using Composer autoloading in context of pure legacy, seeing how you just can do `require autoload.php`, instead of `require vendor/autoload.php` and achieve the exact same thing?

Reply:
@emodric I imagine this allows using composer-installed dependencies in a pure-legacy context... An interesting concept - albeit probably not many legacy extensions are currently available that take advantage of this

Reply:
> albeit probably not many legacy extensions are currently available that take advantage of this

@gggeek If any, hence, my question :) But I suppose there's no harm done in allowing Composer autoloading too.

Reply:
> Btw, is there a point in using Co

As said, this is not the main point of this, just a side benefit/downside however you look at it.